lib/resolve-package-path.d.ts

```diff
@@ -1,12 +1,9 @@
 import Cache = require('./cache');
 import CacheGroup = require('./cache-group');
-declare function _getRealFilePath(realFilePathCache: Cache, filePath: string): string | null;
-declare function _getRealDirectoryPath(realDirectoryPathCache: Cache, directoryPath: string): string | null;
-declare function _findPackagePath(realFilePathCache: Cache, name: string, dir: string): string | null;
```

Comment:
Hmm, I do find it easier to read these as stand alone `declare function` calls. Why move them down inline?